### PR TITLE
NOISSUE - Sync Env Veriables With Docker Deployment

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
1. Add service db env variable to be different from the default env variables. This had already been done for `InfluxDB` for example for timescale readers and writers the db env variable is `envPrefixDB    = "MF_TIMESCALE_"`
2. Fixed Cassandra `httpserverconfig` to be `envPrefixHttp` as the default and `envPrefix` as the alternative
3. Removed `SignRSABits` and `SignHoursValid` from certs as they are not used
4. Remove initialization of `things` `auth` cache in `mqtt` as it is not used
5. Change things grpc config from `MF_THINGS_AUTH_GRPC_` to `MF_THINGS_GRPC_`
6. Changed Cassandra config not to start with prefix `DB_` for example from `DB_CLUSTER` to `CLUSTER` same as Postgres
7. Added env config for `MF_(service)_HTTP_SERVER_CERT` and `MF_(service)_HTTP_SERVER_KEY` as they had not been used

### Which issue(s) does this PR fix/relate to?
No issue

### List any changes that modify/break current functionality
Ensure configs are passed on docker-compose file. There is no change to the functionality of the system

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### Notes
N/A

### CODE REVIEW

**Review by SammyOina — APPROVED** [review]

File: cmd/cassandra-writer/main.go [hunk 1]
```diff
@@ -106,7 +107,7 @@ func main() {
 	// Create new http server
 	httpServerConfig := server.Config{Port: defSvcHttpPort}
 
-	if err := env.Parse(&httpServerConfig, env.Options{Prefix: envPrefix, AltPrefix: envPrefixHttp}); err != nil {
+	if err := env.Parse(&httpServerConfig, env.Options{Prefix: envPrefixHttp, AltPrefix: envPrefix}); err != nil {
```
Comment (arvindh123) [inline cmd/cassandra-writer/main.go]: @rodneyosodo  @dborovcanin We have planned to remove `AltPrefix` everywhere 
Can we remove in this PR

Comment (dborovcanin) [reply cmd/cassandra-writer/main.go]: If we have all the prefixes set correctly now, we can remove it in this PR.

File: docker/.env [hunk 2]
```diff
@@ -69,75 +82,114 @@ MF_EMAIL_PASSWORD=2b0d302e775b1e
...
 ### VERNEMQ
 MF_DOCKER_VERNEMQ_ALLOW_ANONYMOUS=on
 MF_DOCKER_VERNEMQ_LOG__CONSOLE__LEVEL=error
 
 ### CoAP
 MF_COAP_ADAPTER_LOG_LEVEL=debug
+MF_COAP_ADAPTER_HOST=coap-adapter
 MF_COAP_ADAPTER_PORT=5683
+MF_COAP_ADAPTER_SERVER_CERT=
+MF_COAP_ADAPTER_SERVER_KEY=
```
Comment (arvindh123) [inline docker/.env:156]: @rodneyosodo , Are these two variables needed ?
because you have added `MF_COAP_ADAPTER_HTTP_SERVER_CERT`

Comment (rodneyosodo) [reply docker/.env:156]: Yes. They are added already

Comment (arvindh123) [reply docker/.env:156]: @rodneyosodo , These two variables not used, Please double check again and remove them

File: internal/env/parser.go [hunk 3]
```diff
@@ -85,20 +70,16 @@ func parseGrpcConfig(cfg *grpc.Config, altPrefix string, opts ...env.Options) er
 	return nil
 }
 
-func parseServerConfig(cfg *server.Config, altPrefix string, opts ...env.Options) error {
+func parseServerConfig(cfg *server.Config, opts ...env.Options) error {
```
Comment (arvindh123) [inline internal/env/parser.go]: @rodneyosodo , We can remove this function , No more needed

File: internal/env/parser.go [hunk 4]
```diff
@@ -21,55 +21,40 @@ type Options struct {
...
-		return parseServerConfig(cfg, altPrefix, actOpt...)
+		return parseServerConfig(cfg, altOpts...)
 	default:
-		return env.Parse(v, actOpt...)
+		return env.Parse(v, altOpts...)
 	}
 }
 
-func parseGrpcConfig(cfg *grpc.Config, altPrefix string, opts ...env.Options) error {
+func parseGrpcConfig(cfg *grpc.Config, opts ...env.Options) error {
```
Comment (arvindh123) [inline internal/env/parser.go]: @rodneyosodo , We can remove this function , No more needed

File: docker/.env (hunk 2 above)
Comment (rodneyosodo) [reply docker/.env:156]: I have implemented it to use `TLS` if they are provided

**Comment by drasko** [comment]
@arvindh123 is this one OK for merge? Please review and test.

File: docker/.env (hunk 2 above)
Comment (arvindh123) [reply docker/.env:156]: Okay, Got it

**Review by arvindh123 — APPROVED** [review]
LGTM

**Review by arvindh123 — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**